boxsdk/auth/jwt_auth.py

```diff
@@ -360,3 +361,42 @@ def _normalize_rsa_private_key_passphrase(passphrase):
                 .format(passphrase.__class__.__name__)
             )
         return passphrase
+
+    @classmethod
+    def from_settings_dictionary(cls, config_dictionary, **kwargs):
+        """
+        Create an auth instance as defined by the given settings dictionary.
+
+        The dictionary should have the structure of the JSON file downloaded from the Box Developer Console.
+
+        :param config_dictionary:       Dictionary containing settings for configuring app auth.
+        :type config_dictionary:        `dict`
+        :return:                        Auth instance configured as specified by the config dictionary.
+        :rtype:                         :class:`JWTAuth`
+        """
+        if 'boxAppSettings' not in config_dictionary:
+            raise ValueError('boxAppSettings not present in configuration')
+        return cls(
+            client_id=config_dictionary['boxAppSettings']['clientID'],
+            client_secret=config_dictionary['boxAppSettings']['clientSecret'],
+            enterprise_id=config_dictionary.get('enterpriseID', None),
+            jwt_key_id=config_dictionary['boxAppSettings']['appAuth'].get('publicKeyID', None),
+            rsa_private_key_data=config_dictionary['boxAppSettings']['appAuth'].get('privateKey', None),
+            rsa_private_key_passphrase=config_dictionary['boxAppSettings']['appAuth'].get('passphrase', None),
+            **kwargs
+        )
+
+    @classmethod
+    def from_config_file(cls, config_file_sys_path, **kwargs):
```

Comment:
Named now mismatch. Can you make this one `from_settings_*` as well?

And can you name the params for both methonds `settings_*`?